echo-web/src/main/java/com/netflix/spinnaker/echo/config/Front50Config.java

```diff
@@ -64,13 +65,17 @@ public Endpoint front50Endpoint(@Value("${front50.base-url}") String front50Base
 
   @Bean
   public Front50Service front50Service(
-      Endpoint front50Endpoint, OkHttpClient okHttpClient, LogLevel retrofitLogLevel) {
+      Endpoint front50Endpoint,
+      OkHttpClient okHttpClient,
+      LogLevel retrofitLogLevel,
+      RequestInterceptor spinnakerRequestInterceptor) {
```

Comment:
This should be `SpinnakerRequestInterceptor`... what we want is not just _any_ `RequestInterceptor`, but rather a very specific one.

Reply:
Looks like you probably copied this from `IgorConfig.java`... could you fix it there too?

Reply:
Currently, there is only one implementation of `RequestInterceptor` which is `SpinnakerRequestInterceptor`. So this configuration will always pick up `SpinnakerRequestInterceptor`. 

But I do agree with your point of making it specific, so in future, there will be no conflict and indeed all this configuration need is `SpinnakerRequestInterceptor`. So will address your comment. Thanks for pointing it out.